docs/resources/logdna_ingestion_exclusion.md

```diff
@@ -0,0 +1,39 @@
+# Resource: `logdna_ingestion_exclusion`
+
+The resource allows you to filter out logs that you don't need to store, preventing lines from being ingested
+in our searchable database. You can define exclusion rules by application, hostname, and patterns within log
+lines.
+
+## Example
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+}
+
+resource "logdna_ingestion_exclusion" "http-success" {
+  title  = "HTTP 2XX"
+  apps   = ["nginx", "apache"]
+  query  = "response:(>=200 <300) request:*"
+  active = true
+}
+
+resource "logdna_ingestion_exclusion" "http-noise" {
+  title  = "Noisy HTTP Paths"
+  apps   = ["nginx", "apache"]
+  query  = "robots.txt OR favicon.ico OR .well-known"
+  active = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported by `logdna_ingestion_exclusion`:
+
+_Note:_ A `title` and at least one of the following properties: `apps`, `hosts`, `query` must be specified to create this resource.
+
+- `title`: **string** _(Required)_ Title of this exclusion rule that will appear in the UI.
```

Comment:
already called out by @david-ly in the code, just noting this is still marked required in docs as well